app/web/src/features/donations/Donations.tsx

```diff
@@ -128,15 +117,17 @@ const useStyles = makeStyles((theme) => ({
 export default function Donations() {
   const classes = useStyles();
 
+  const { t } = useTranslation("donations");
+
   return (
     <>
-      <HtmlMeta title={DONATE} />
+      <HtmlMeta title={t("donate")} />
```

Comment:
Do these keys autocomplete?

Reply:
It does after adding augmenting the TypeScript types mentioned in the docs!